basics/getting_started.md

```diff
@@ -114,7 +114,7 @@ However, to debug Java code, attach to the server using the following:
 *  Run Bazel with the debugging option `--host_jvm_debug` before the
    command (e.g., `bazel --host_jvm_debug build //src:bazel`).
 *  Attach a debugger to the port 5005. For instance, with `jdb`,
-   run `jdb -attach localhost:5005`.
+   run `jdb -attach localhost:5005` (`jdb.exe -connect com.sun.jdi.SocketAttach:hostname=localhost,port=5005` for Windows).
```

Comment:
Can you reformat this so it is easier to read.  More like

Attach a debugger ....
  linux: run jdb  -attach localhost:5005
  windows: jdb.exe -connect com.sun.jdi.SocketAttach:hostname=localhost,port=5005`

Reply:
Sure!
I have reformatted the text.

Reply:
thanks